src/components/Button/index.tsx

```diff
@@ -7,7 +7,7 @@ import { RowBetween } from '../Row'
 
 type ButtonProps = Omit<ButtonPropsOriginal, 'css'>
 
-export const BaseButton = styled(RebassButton)<
+const BaseButton = styled(RebassButton)<
```

Comment:
this is awesome, impossible to spot without a linter, can be confusing to see exported members while refactoring